src/System.Private.Windows.Core/src/Windows/Win32/UI/Controls/Dialogs/DEVNAMES.cs

```diff
@@ -8,7 +8,9 @@ namespace Windows.Win32.UI.Controls.Dialogs;
 /// </summary>
 /// <remarks>
 ///  <para>
-///   <see href="https://learn.microsoft.com/windows/win32/api/commdlg/ns-commdlg-devnames">Learn more about this API from learn.microsoft.com</see>.
+///   <see href="https://learn.microsoft.com/windows/win32/api/commdlg/ns-commdlg-devnames">
+///    Learn more about this API from learn.Microsoft.com
+///   </see>.
```

Comment:
I like this split here. Let's put the period within the content.
```suggestion
///   <see href="https://learn.microsoft.com/windows/win32/api/commdlg/ns-commdlg-devnames">
///    Learn more about this API from learn.microsoft.com.
///   </see>
```

Reply:
There are a lot of those I agree I will fix them all. I will leave Microsoft as is until I hear from you.